src/main/java/de/ellpeck/rockbottom/api/construction/compendium/construction/ConstructionRecipe.java

```diff
@@ -47,16 +47,18 @@
     protected final List<IUseInfo> inputs;
     protected final List<ItemInstance> outputs;
     protected final List<ConstructionTool> tools;
+    protected final boolean constructionTable;
```

Comment:
I would change this to be "manual" or "manualOnly" and then change the "showInConstructionTable" to be "!manual"